src/Checkout/Plus/PlusPaymentFinalizeController.php

```diff
@@ -49,7 +49,7 @@ public function __construct(
 
     /**
      * @RouteScope(scopes={"storefront"})
-     * @Route("/paypal/plus/payment/finalize-transaction", name="paypal.plus.payment.finalize.transaction", methods={"GET"}, defaults={"auth_required"=false})
```

Comment:
changing the name is actually a break, isn't it? I think we have to revert this

Reply:
I don't now, is it?
But without this change I have no idea how to fix the bug.
Is adding a second route with the same path but another name possible?

Reply:
ah I see. didn't read your last sentence properly :see_no_evil: 

then I see no other way and we will have to do the break :man_shrugging: